src/modules/songs/components/SongNotes/SongNotes.tsx

```
@@ -0,0 +1,140 @@
+    <SongDetailsItem icon="NotePencil" label="Notes">
+      {!isEditingNotes ? (
+        <div
+          className={cn("hover:cursor-pointer", { "p-[9px]": !isEditingNotes })}
+          onClick={() => setIsEditingNotes(true)}
+          onKeyDown={(e) => {
+            if (e.key === "Enter" || e.key === " ") {
+              e.preventDefault();
+              setIsEditingNotes(true);
+            }
+          }}
+          role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)